cmd/integrationArtifactUpload_test.go

```diff
@@ -26,7 +26,7 @@ func TestRunIntegrationArtifactUpload(t *testing.T) {
 
 	t.Run("Successfull Integration Flow Create Test", func(t *testing.T) {
 		filesMock := mock.FilesMock{}
-		path := filepath.Join("tempDir", "iflow4.zip")
+		path := filepath.Join("tempDir", "integrationFlow4.zip")
```

Comment:
I wasn't sure about these 4- should I be changing the name of the zip? IT seems like this zip is pulled from somewhere.

Reply:
I don't think it is downloaded. If the file does not exist in this repo then it is probably just a mock value. If the tests don't fail all is good, keep the change.

Reply:
The tests don't fail, so I'll keep it then.